### ISSUE TITLE
[FEAT] Publish to npm

### ISSUE THREAD

**Issue by strugee** [mikejgray/ovos-skill-projen#7]
### Objective

It would be great if I could just `npm install` this without mucking about with third-party registries.

The config file editing is mildly annoying. The real annoyance, however, is that I have to authenticate to GitHub before it'll let me install packages: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-to-github-packages

This raises the barrier to entry a decent amount and IMO may be enough to discourage drive-by skill writing by people who might be interested if we're able to pull them into the Mycroft community.

### Initial Implementation Requirements

 - `npm publish`
 - profit

**Comment by strugee** [comment]
@mikejgray thanks! In case you didn't see though, the workflow failed: https://github.com/mikejgray/ovos-skill-projen/actions/runs/5709697090/job/15468937331

**Comment by mikejgray** [comment]
Yep, although I'm not sure how to fix it at this point, so I've manually published 0.0.14 to npm for now. I'll keep this issue open until I've got an automated fix. Thanks for the callout!

**Comment by strugee** [comment]
Thank you! I see it on the registry, much appreciated.
